entrypoint.sh

```diff
@@ -1,42 +1,48 @@
 #!/bin/bash
 
-dataset_f=$1
-participant_f=$2
-out=${3-.}
-jobs=${4-6}
+machine_f=$1
+out=${2-.}
```

Comment:
for clarity, what about:
```suggestion
output_dir=${2-.}
```